emscripten.py

```diff
@@ -874,6 +874,17 @@ def get_exported_implemented_functions(all_exported_functions, all_implemented,
     if key in all_exported_functions or export_all or (export_bindings and key.startswith('_emscripten_bind')):
       funcs.add(key)
 
+  if not export_all:
+    aliases = metadata['aliases'].keys()
+    for key in all_exported_functions:
```

Comment:
Is this outer loop needed? (`all_exported_functions` may be very large, which worries me)

Instead, how about
```
for name, alias in metadata['aliases']:
  if name in all_implemented:
    funcs.add(alias)
```

Reply:
thats is actually to reduce the number of exports. we could just export all the alias like what you did, i think there arent many.

Reply:
thats also why its only run if not export_all, so there shouldnt be that many exports i hope.